### ISSUE TITLE
login not working 

### ISSUE THREAD

**Issue by hossnys** [threefoldtech/taiga-contrib-threefold-auth#3]
we still get this error when trying login 

![image](https://user-images.githubusercontent.com/26298009/186423938-df8f525e-8402-42b7-b8ed-930afcc07baf.png)

and when checking taiga-back logs found this error happen every time trying login :

```
taiga-back_1             | Executing pending migrations
taiga-back_1             | Operations to perform:
taiga-back_1             |   Apply all migrations: admin, attachments, auth, bitbucket, contact, contenttypes, custom_attributes, djmail, easy_thumbnails, epics, external_apps, feedback, github, gitlab, gogs, history, issues, likes, milestones, notifications, projects, references, sessions, settings, tasks, telemetry, timeline, token_denylist, users, userstorage, userstories, votes, webhooks, wiki
taiga-back_1             | Running migrations:
taiga-back_1             |   No migrations to apply.
taiga-back_1             | Load default templates
taiga-back_1             | Installed 2 object(s) from 1 fixture(s)
taiga-back_1             | Give permission to taiga:taiga
taiga-back_1             | Starting Taiga API...
taiga-back_1             | [2022-08-24 12:48:20 +0000] [1] [INFO] Starting gunicorn 19.9.0
taiga-back_1             | [2022-08-24 12:48:20 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
taiga-back_1             | [2022-08-24 12:48:20 +0000] [1] [INFO] Using worker: sync
taiga-back_1             | [2022-08-24 12:48:20 +0000] [29] [INFO] Booting worker with pid: 29
taiga-back_1             | [2022-08-24 12:48:20 +0000] [30] [INFO] Booting worker with pid: 30
taiga-back_1             | [2022-08-24 12:48:20 +0000] [31] [INFO] Booting worker with pid: 31
taiga-back_1             | [2022-08-24 12:49:53 +0000] [1] [CRITICAL] WORKER TIMEOUT (pid:29)
taiga-back_1             | [2022-08-24 12:49:54 +0000] [44] [INFO] Booting worker with pid: 44
taiga-back_1             | [2022-08-24 12:52:47 +0000] [1] [CRITICAL] WORKER TIMEOUT (pid:30)
taiga-back_1             | [2022-08-24 12:52:47 +0000] [49] [INFO] Booting worker with pid: 49


```
and this [fix](https://github.com/threefoldtech/tf-images/pull/81) didn't help there

**Comment by sameh-farouk** [comment]
I tried to curl the backend directly and isolate any issue that could be related to the reverse proxy configuration.
It seems the backend will stuck at something when handling callback from the auth server. The gunicorn web server, which handle the python app, will drop the request as it timed out.
Checking auth code to guess where the request could be stuck shows that the handler will send a post request to `https://openkyc.live/verification/verify-sei` as a part of the auth verification.
So I tried to access this URL from inside the container, and found that the container has internet connectivity issue.
So I suspect that it is a docker/ docker network issue, maybe bad setup or privilege issue.
